Angular.md

```diff
@@ -3,8 +3,19 @@
 ## For Beginners
 
 1.  [Angular 2 Book](https://goalkicker.com/Angular2Book/)
-1.  [Angular 6 Complete Playlist (45 videos)](https://www.youtube.com/playlist?list=PLC3y8-rFHvwhBRAgFinJR8KHIrCdTkZcZ)
+2.  [Angular 6 Complete Playlist (45 videos)](https://www.youtube.com/playlist?list=PLC3y8-rFHvwhBRAgFinJR8KHIrCdTkZcZ)
 
 ## For People with Intermediate Level Expertise
 
+### RXJS - Reactive Programming
```

Comment:
Reactive programming is a general topic. Please make a separate file dedicated to Reactive programming and add there.